### DISCUSSION TITLE
No Playername Formatting in Console/Logs

### DISCUSSION THREAD

**Discussion by digitalcrowd** [Zrips/CMI#3939] [Feature Requests]
Since {#color} has been added, player names show up wonky in console and in logs. This makes it hard to easily search the logs with "grep", but having to find out how their player name is formatted before gripping.

```
[14:05:02 INFO]:  §xJ§xu§xs§xt§xK§xa§xr§xm§xa joined the game
[14:05:06 INFO]: [§xDRFTR] PupsDaArtist: eyy
[14:05:08 INFO]: [§xDRFTR] Emily463: ayyy
[14:05:19 INFO]: [§xLEGND] §xJ§xu§xs§xt§xK§xa§xr§xm§xa: aye
```

It would be nice if no formatting was sent to console and logs. Thanks!

**Comment by CloudeLecaw** [comment]
This normal because this is how it works internaly they give a colorcode with 2letters like xX and that is telling them what hexcode is used. You console just shows the true colorcodes behind the hexcolor magic. ;) by the way... Look at plugins that does nit Support this like the console itself or dynmap there a hex colored name looks totally different 😂
There is nothing cmi can do here. Its your console that must be able to interprete these colored nicknames. Also cmi (minecraft maybe?) itself dies not fully support this. In chat all in fine..but try open a home gui.. 😂 Or some other guis where you name is used. Even on nickname command your name looks strengh because it uses the real colorcodes.. I think there are many many steps before owner may should complain about the console view of rainbow nicks ;-)

Finaly to say, its no bug and nothing zrips can do with cmi here at the moment because cmi dont touches your console or logs.  For what you want cmi would need a own log and console app that respects the colorformats of the hex Magic and dont translate them to their true colorcode
